### PR TITLE
locators 4.14 provided

### PR DESCRIPTION
**Pull request by DanielOsypenko**

### CODE REVIEW

**Review by petr-balogh — DISMISSED** [review]

**Comment by DanielOsypenko** [comment]
the overview title has changed. Bug is opened - https://issues.redhat.com/browse/OCPBUGS-15419

**Comment by DanielOsypenko** [comment]
> PR Test Path: tests/ui/test_non_admin_user.py::TestOBCUi::test_create_storageclass_rbd

resource leftover. Will be rerun.

File: ocs_ci/ocs/ui/views.py [hunk 1]
```diff
@@ -18,13 +18,17 @@
...
-    "skip_tour": (By.CSS_SELECTOR, 'button[data-test="tour-step-footer-secondary"]'),
+    "skip_tour": ('button[data-test="tour-step-footer-secondary"]', By.CSS_SELECTOR),
 }
 
 login_4_11 = {
     "ocp_page": "Overview · Red Hat OpenShift",
     "login_page_title": "Log in · Red Hat OpenShift",
 }
+# Bug opened in Jira https://issues.redhat.com/browse/OCPBUGS-15419. Tmp solution to check locators
+login_4_14 = {
```
Comment (OdedViner) [inline ocs_ci/ocs/ui/views.py:29]: Can you move it to https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/ui/views.py
?

**Review by OdedViner — DISMISSED** [review]

File: ocs_ci/ocs/ui/base_ui.py [hunk 2]
```diff
@@ -55,6 +56,54 @@
...
+        timeout (int): Maximum time (in seconds) to wait for the element to be clickable. Defaults to 30 seconds.
+
+    Returns:
+        selenium.webdriver.remote.webelement.WebElement: The clickable web element.
+
+    """
+    wait = WebDriverWait(SeleniumDriver(), timeout)
+    try:
+        web_element = wait.until(
+            ec.visibility_of_element_located((locator[1], locator[0]))
```
Comment (ebenahar) [inline ocs_ci/ocs/ui/base_ui.py:74]: shouldn't we use `element_to_be_clickable()` instead of `visibility_of_element_located()`?

**Review by ebenahar — APPROVED** [review]

**Review by petr-balogh — APPROVED** [review]

**Review by petr-balogh — APPROVED** [review]

**Review by OdedViner — APPROVED** [review]

**Pull request merged**